bundles/org.openhab.ui/doc/components/oh-plan-marker.md

```diff
@@ -49,7 +49,7 @@ A marker on a floor plan
 ### Icon
 <div class="props">
 <PropGroup name="icon" label="Icon">
-  You can customize the styles further with CSS attributes in the <code>iconStyle</code> parameter (in YAML only)
+  You can customize the styles further with CSS attributes in the <code>iconStyle</code> parameter (in YAML only) - see example below
```

Comment:
Again, these are generated from the config properties descriptions so you can't add anything, it will be overwritten when these parts of the docs are regenerated.